indexUtil.sh

```diff
@@ -127,6 +131,7 @@ function buildUpdateByQueryUrl() {
     local _env=$2
     local _idxName=$3
     local _docType=$4
+    local _slices=$5
```

Comment:
where are you using _slices ?

Reply:
`updateByQueryLongRunning.sh` and only there. I noted that in the README, and also noted it was required in the comment header on that file.